server/models/request.model.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+
+const requestSchema = mongoose.Schema({
+    user_id: {
+        type: String,
+        required: true
+      },
+    sitter_id: {
+        type: String,
+        required: true
+      },
```

Comment:
Please do camelCase and not snake_case, JS Convention. Also, referencing the model would have been ideal here as opposed to just strings.